tamr_toolbox/utils/operation.py

```diff
@@ -187,3 +187,18 @@ def monitor(
         else:
             return operation
     raise TimeoutError(f"Waiting for operation took longer than {timeout_seconds} seconds.")
+
+
+def safe_estimate_counts(project) -> Operation:
+    """
+    Run the estimate counts job of project that works if it is the first
+    Args:
+        project: A Tamr project object
+    Returns:
+        An operation object for the estimate pairs job
+    """
+    project = project.as_mastering()
+    response = project.client.post(
+        f"/api/versioned/v1/projects/{project.resource_id}/estimatedPairCounts:refresh"
```

Comment:
When using the `client.post` method, I don't believe you need to include `/api/versioned/v1/` part of the url, since this is the default `base_path`. You can also get `f"projects/{project.resource_id}"` from the `api_path` attribute:
```suggestion
        f"{project.api_path}/estimatedPairCounts:refresh"
```